apps/vr-tests/src/utilities/steps.ts

```diff
@@ -0,0 +1,14 @@
+import { Steps as OriginalSteps } from 'screener-storybook/src/screener';
+
+OriginalSteps.prototype.setRTL = function (isRTL: boolean = false) {
+
+  var step = {
+    type: 'executeScript',
+    code: `document.documentElement.setAttribute("dir", ${isRTL ? '"rtl"' : '"ltr"'})`
```

Comment:
Doesn't `getRTL()` in utilities cache this value?

Reply:
yes. If this is a page refresh, then cool. but we can't simply flip and expect things to work without a re-render.

You can call setRTL in utilities to change it and re-render

Reply:
ah, good point. It was working for my single example, so maybe I need to look at using getRTL from fabric utilities

Reply:
Do I need to create a public method on a wrapping component, and then call it via a reference to the component? Or is there an easier way?